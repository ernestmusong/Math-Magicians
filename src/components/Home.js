import React from 'react';
import { LoremIpsum } from 'react-lorem-ipsum';

function Home() {
  return (
    <section className="home">
      <div className="contentWrap">
        <h2>Welcome to our page!</h2>
        <p><LoremIpsum p={2} /></p>
      </div>
    </section>
  );
}

export default Home;
