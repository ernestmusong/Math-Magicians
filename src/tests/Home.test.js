import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

describe('Navbar', () => {
  it('test Header rendering', () => {
    render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    );
    const headerElement = screen.getByText(/Welcome to our page!/i);
    expect(headerElement.textContent).toBe('Welcome to our page!');
  });
});
