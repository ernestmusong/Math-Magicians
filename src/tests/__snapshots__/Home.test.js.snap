// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the home component 1`] = `
<section
  className="home"
>
  <div
    className="contentWrap"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      <p>
        Lorem ipsum odor amet, consectetuer adipiscing elit. Hendrerit lacus duis hac augue curabitur. Turpis elit consectetur; amet sociosqu in nullam. Pulvinar ipsum ante blandit pharetra, id magnis aliquet ipsum. Consectetur fermentum inceptos dapibus sollicitudin metus ipsum. Nisl potenti pulvinar quam elementum integer risus inceptos condimentum? Curae penatibus nullam quis primis mauris neque porttitor ipsum aptent. Class lacinia velit ullamcorper habitasse sagittis ex nunc integer. Massa tempus placerat iaculis at per quisque efficitur suscipit. Vehicula tellus tellus amet taciti molestie lobortis.
      </p>
      <p>
        Enim condimentum quisque blandit consectetur aptent; mattis dui. Lobortis volutpat id senectus vel egestas; primis rutrum molestie. Commodo mus eleifend dignissim fermentum consectetur imperdiet fermentum torquent inceptos. Diam tempor nisl aenean et suspendisse tempor in. Himenaeos nec est feugiat duis in risus. Dis rutrum dictum ipsum suscipit convallis habitant primis nascetur. Ultrices parturient nisl hac ex magnis nibh. Mattis sollicitudin scelerisque conubia placerat blandit bibendum mattis placerat.
      </p>
    </p>
  </div>
</section>
`;
